examples/hpc/dla-driver/src/tensor3.rs

```diff
@@ -0,0 +1,248 @@
+#![no_std]
+#![no_main]
+
+use alloc::vec::*;
+use headsail_bsp::ufmt::uDisplay;
+use headsail_bsp::{sprint, sprintln};
+use ndarray::{Array, Array3};
+
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum Order3 {
+    CHW,
+    CWH,
+    HWC,
+    HCW,
+    WHC,
+    WCH,
+}
+
+impl From<Order3> for [usize; 3] {
+    fn from(order: Order3) -> Self {
+        match order {
+            Order3::CHW => [0, 1, 2],
+            Order3::CWH => [0, 2, 1],
+            Order3::HWC => [1, 2, 0],
+            Order3::HCW => [1, 0, 2],
+            Order3::WHC => [2, 1, 0],
+            Order3::WCH => [2, 0, 1],
+        }
+    }
+}
```

Comment:
`From`/`Into` is for 1:1 type conversions. If converting to a specific kind of representation it may be more clear to use something like `Order3::as_positions` / `into_positions`. This allows for multiple implementations, e.g. `fn into_tensorflow_order`, `fn into_reverse_tensorflow_order`, `fn into_custom_order`.

Reply:
Moved this functionality to `into_position` function